packages/components/Avatar/src/useAvatar.ts

```diff
@@ -76,6 +77,11 @@ export const useAvatar = (props: AvatarProps): AvatarInfo => {
     };
   }
 
+  const disableBadge = Platform.OS === 'android' && size === 16;
+  if (disableBadge) {
+    state.showBadge = false;
+  }
```

Comment:
How did you check for ring removal?

Reply:
Ring removal not required as per design guidelines